rust/src/pgsql/pgsql.rs

```diff
@@ -134,6 +142,7 @@ pub struct PgsqlState {
     backend_pid: u32,
     state_progress: PgsqlStateProgress,
     tx_index_completed: usize,
+    events: u16,
```

Comment:
This field seems unused

Reply:
Not sure if this makes sense or not, but... The implementation I mostly see, the events are set in the transactions. But sometimes, there may not be a transaction, yet. Would it make any sense to have this counter here, for such cases, at least?

Reply:
Maybe it makes sense at some point to have this counter, but the code now does not make use of it

So, I would remove it until it is really needed